drdown/users/locale/pt_BR/LC_MESSAGES/django.po

```diff
@@ -81,76 +81,76 @@ msgstr "Acrônimo do Conselho"
 
-#: drdown/users/models/model_health_team.py:119
+#: drdown/users/models/model_health_team.py:120
 msgid "The registration state of member of health team."
 msgstr "O estado de registro do membro da equipe médica."
 
-#: drdown/users/models/model_health_team.py:123
-#: drdown/users/models/model_health_team.py:134
+#: drdown/users/models/model_health_team.py:124
+#: drdown/users/models/model_health_team.py:135
 msgid "Speech Therapy"
 msgstr "Fonoaudiologia"
 
-#: drdown/users/models/model_health_team.py:124
-#: drdown/users/models/model_health_team.py:135
+#: drdown/users/models/model_health_team.py:125
+#: drdown/users/models/model_health_team.py:136
 msgid "Psychology"
 msgstr "Psicologia"
 
-#: drdown/users/models/model_health_team.py:125
-#: drdown/users/models/model_health_team.py:136
+#: drdown/users/models/model_health_team.py:126
+#: drdown/users/models/model_health_team.py:137
 msgid "Physiotherapy"
 msgstr "Fisioterapia"
 
-#: drdown/users/models/model_health_team.py:126
-#: drdown/users/models/model_health_team.py:137
+#: drdown/users/models/model_health_team.py:127
+#: drdown/users/models/model_health_team.py:138
 msgid "Occupational Therapy"
 msgstr "Terapia Ocupacional"
 
-#: drdown/users/models/model_health_team.py:127
-#: drdown/users/models/model_health_team.py:138
+#: drdown/users/models/model_health_team.py:128
+#: drdown/users/models/model_health_team.py:139
 msgid "Doctor"
 msgstr "Médico"
 
-#: drdown/users/models/model_health_team.py:128
-#: drdown/users/models/model_health_team.py:139
+#: drdown/users/models/model_health_team.py:129
+#: drdown/users/models/model_health_team.py:140
 msgid "Cardiology"
 msgstr "Cardiologia"
 
-#: drdown/users/models/model_health_team.py:129
-#: drdown/users/models/model_health_team.py:140
+#: drdown/users/models/model_health_team.py:130
+#: drdown/users/models/model_health_team.py:141
 msgid "Neurology"
 msgstr "Neurologia"
 
-#: drdown/users/models/model_health_team.py:130
-#: drdown/users/models/model_health_team.py:141
+#: drdown/users/models/model_health_team.py:131
+#: drdown/users/models/model_health_team.py:142
 msgid "Pediatrics"
 msgstr "Pediatria"
 
-#: drdown/users/models/model_health_team.py:131
-#: drdown/users/models/model_health_team.py:142
-#| msgid "Nursery"
+#: drdown/users/models/model_health_team.py:132
+#: drdown/users/models/model_health_team.py:143
 msgid "Nursing"
 msgstr "Enfermagem"
 
-#: drdown/users/models/model_health_team.py:146
+#: drdown/users/models/model_health_team.py:147
 msgid "Speciality"
 msgstr "Especialidade"
 
-#: drdown/users/models/model_health_team.py:148
+#: drdown/users/models/model_health_team.py:149
 msgid "The speciality that this member of health team works."
 msgstr "A especialidade do membro da equipe médica."
 
-#: drdown/users/models/model_health_team.py:208
+#: drdown/users/models/model_health_team.py:209
 #, python-format
 msgid "A %(speciality)s cannot have a %(register)s"
 msgstr "%(speciality)s não pode ter %(register)s"
```

Comment:
Eu trocaria aqui a msgid e a msgstr:
msgid "The %(register)s doesn't inscribe professionals with %(speciality)s graduation, please correct"
msgstr "O %(register)s não registra profissionais com formação em %(speciality)s, favor corrigir"

Obs: Meu inglês não é perfeito, então se puderem validar!

Reply:
![image](https://user-images.githubusercontent.com/31926918/39205742-11a47ff8-47eb-11e8-8a32-bb47b2340045.png)